packages/@sanity/base/src/change-indicators/ChangeBar.styled.tsx

```diff
@@ -0,0 +1,211 @@
+import React from 'react'
+import styled, {css} from 'styled-components'
+import {Theme} from '@sanity/ui'
+import {EditIcon} from '@sanity/icons'
+
+interface BarAndShapeInterface {
+  theme: Theme
+}
+
+interface RootInterface {
+  focus: boolean
+  hover: boolean
+  changed: boolean
+  isReviewChangeOpen: boolean
+  disabled: boolean
+}
+
+const Shape = (props: Omit<React.SVGProps<SVGElement>, 'ref'>) => {
+  return (
+    <svg {...props} viewBox="0 0 20 27" fill="none" xmlns="http://www.w3.org/2000/svg">
+      <path
+        d="M9 0.448608C9 2.49663 7.38382 4.13678 5.57253 5.09261C2.55605 6.68443 0.5 9.8521 0.5 13.5C0.5 17.1479 2.55605 20.3155 5.57253 21.9074C7.38382 22.8632 9 24.5033 9 26.5514V27H11V26.5514C11 24.5033 12.6162 22.8632 14.4275 21.9074C17.4439 20.3155 19.5 17.1479 19.5 13.5C19.5 9.8521 17.4439 6.68443 14.4275 5.09261C12.6162 4.13678 11 2.49663 11 0.448608V0H9V0.448608Z"
+        fill="currentColor"
+      />
+    </svg>
+  )
+}
+
+const animationSpeed = 150
+
+export const responsiveSizes = {
+  maxScreenMedium: `(max-width: 511px)`,
```

Comment:
are these breakpoints something we have defined in a guide or somewhere? would be good to add link to docs for the next reader (if we have).

Reply:
These are values that we were using as variables before, can be seen in the `responsive.css` file, but because we don't have any. Not necessarily something that in sanity ui I simply used the same values. I'll double check with Marius 🤔

Reply:
If nothing else I can add a note related to this in the comments of the code itself